### ISSUE TITLE
Course overview headings

### ISSUE THREAD

**Issue by Kilavagora** [Kilavagora/JavaScript-Class#1]
## Issue

As you can see, [course outline](/Kilavagora/JavaScript-Class/blob/master/README.md) is rather plain looking. It would definitely benefit if it had proper headings for the sections. JavaScript course takers would also be happier to see the pretty looking outline, that would increase their motivation.

## Proposed improvement

The title _JavaScript Course Outline_ needs to be marked as heading level 1. _Debut_, _Mittelspiel_, _Zugzwang_ and _Endspiel_ should be heading level 2. The rest of the headings should be heading level 3.

## Instructions

* I
  * Fork this repository
  * ~Mark your fork as private~
* II
  * Add me as a colaborator to your fork
* III
  * Create a branch for your changes
  * Commit your changes to your branch to improve the markdown of README.md file according to the proposed improvement above
* IV
  * Create a pull request to your master ~and mark me as a reviewer~
* I-II-III-IV
  * Ask questions :)

### Short Glossary

* *Git* - Git is used for managing a project, or a set of files, as they change over time; Also to coordinate work on those files among multiple people.
* *Fork* - Forking a project means to make its copy.
* *Pull Request* - Propose changes to a branch or someone else's project.
* *Markdown* - is a lightweight markup language to format plain text.

```
3e7c279c4a5316a7fd60add84fc833c8  README.md
```

**Comment by Kilavagora** [comment]
Colin pointed out that you cannot mark fork of a public repo as private. So crossed that out.

**Comment by Kilavagora** [comment]
If I did not accept your collaboration request on time, you might not see the option to add me as a reviewer. Crossing that out.

**Comment by Kilavagora** [comment]
Please make pull requests to **your own master** branch.

**Comment by Kilavagora** [comment]
Use markdown syntax. Html does not count :)

**Comment by Rich-Sales007** [comment]
I believe I accomplished all the steps, however was I supposed to attach a file when creating the pull request?

**Comment by Kilavagora** [comment]
@Rich-Sales007 No, point of git and pull requests is that you don't have to attach anything :)

**Comment by marcopolo316** [comment]
I have finished the assignment, how does everything look?

**Comment by Kilavagora** [comment]
@marcopolo316 you need to make pull request to your master branch, not here.
